src/DetailsView/reports/components/report-sections/details-section.tsx

```diff
@@ -21,7 +27,10 @@ export const DetailsSection = NamedSFC<DetailsSectionProps>('DetailsSection', pr
                         <td className="icon" aria-hidden="true">
                             <UrlIcon />
                         </td>
-                        <td className="text">
+                        <span className="screen-reader-only" id="target-page-text">
+                            {screenReaderTexts.targetPageLink}
+                        </span>
+                        <td className="text" aria-labelledby="target-page-text">
```

Comment:
both NVDA and JAWS are reading this fields twice, one for the span `screen-reader-only` and then again for the td (minus the 'target page', 'scan date', 'comment' text)
I guess you may need to use aria-hidden on this td's and extensively test with both screen readers.

Reply:
I saw it with NVDA and it was not reading twice. Maybe need to check twice

Reply:
I dont see NVDA repeating twice but JAWS certainly do read the whole table at once.